ramls/inventory-hierarchy.raml

```diff
@@ -0,0 +1,70 @@
+#%RAML 1.0
+title: Inventory Hierarchy API
+version: v1.10
```

Comment:
Please update version here to match module descriptor.

Reply:
Done